launch/launch/launch_service.py

```diff
@@ -67,6 +68,9 @@ def __init__(
         """
         # Setup logging and debugging.
         launch.logging.launch_config.level = logging.DEBUG if debug else logging.INFO
+        # Set the log file name.
+        if log_file_name is not None:
+            launch.logging.launch_config.file_name = log_file_name
```

Comment:
how about applying this to logger name line 78 as well?

```suggestion
            # Ensure the log file name ends with `.log`
            if not log_file_name.endswith('.log'):
                log_file_name += '.log'
            launch.logging.launch_config.file_name = log_file_name
```

Reply:
suggestion applied